docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.4'
+
+services:
+  app:
+    build: .
+    environment:
+      RACK_ENV: development
```

Comment:
Do we need `RAILS_ENV` or will Rails pick up `RACK_ENV` as well?

Reply:
Looks like Rails will pick it up:
https://github.com/rails/rails/blob/master/railties/lib/rails.rb#L73